episodes/administer_hub.md

```diff
@@ -47,6 +47,58 @@ Fill in the blanks with the following words: *organization, team, maintainers, o
 
 ::::
 
+::::{dropdown} Optimise server resources for your work
+:open:
+
+In each scenario, select the most suitable option that makes best use out of the following server resources: *Shared Small: 1–4 CPU, 8–32 GB, Small: 4 CPU, 32 GB, Medium: 16 CPU, 128 GB, Large: 64 CPU, 512 GB*.
+
+a. Run a software pipeline that has a single-thread task requiring 2GB of RAM and 60 tasks in total.
+
+b. Create documentation and edit text files.
+
+c. Compile a software program that ran out of memory and failed after initially building on a *Shared Small* machine.
```

Comment:
This one feels a little unclear, perhaps because there is no "correct" answer that exactly fits the framing of the exercise. Perhaps you could add it as a "stretch goal" question after the first three. E.g.

> Ruth wants to run a new program but does not know its maximum memory requirement. She tried compiling it a *Shared Small* machine, where it initially built but then ran out of memory and failed on execution. Which server configuration should Ruth choose to try to run the program again?

Reply:
I like this idea! Implemented in https://github.com/czi-catalystproject/hub-champion-training/pull/26/commits/7e9eacea83202d44f80bf0e1b25d4d3cdd48cb92